binderhub/templates/page.html

```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
-<html>
+<html lang="en"> 
+  <!-- adding language description for accessibility  -->
```

Comment:
This comment can be part of the pull request description or a commit message adding it, but I think its better omitted from the HTML code itself.

Below is a code suggestion to remove it using the GitHub user interace!

```suggestion
```